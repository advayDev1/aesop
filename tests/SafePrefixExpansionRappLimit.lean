/-
Copyright (c) 2024 Jannis Limperg. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Jannis Limperg
-/

import Aesop

@[aesop safe]
axiom loopy {α : Prop} : α ∨ α → α

example : false := by
  aesop
