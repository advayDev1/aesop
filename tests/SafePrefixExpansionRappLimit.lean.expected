tests/SafePrefixExpansionRappLimit.lean:13:2: warning: aesop: safe prefix was not fully expanded because the maximum number of rule applications (50) was reached.
tests/SafePrefixExpansionRappLimit.lean:13:2: warning: aesop: failed to prove the goal. Some goals were not explored because the maximum rule application depth (30) was reached. Set option 'maxRuleApplicationDepth' to increase the limit.
tests/SafePrefixExpansionRappLimit.lean:12:19: error: unsolved goals
case a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a.a
‚ä¢ False
